pkg/utils/k8sutil.go

```diff
@@ -171,14 +123,14 @@ func GetRestClient() (*rest.RESTClient, error) {
 	return restClient, nil
 }
 
-// GetTPRClient returns tpr client to the request from inside of cluster
-func GetTPRClient() (*rest.RESTClient, error) {
+// GetCRDClient returns tpr client to the request from inside of cluster
+func GetCRDClient() (*rest.RESTClient, error) {
```

Comment:
These functions were always terrible, and impossible to add tests around since they create the config *and* the client.

If we're going to touch these - I suggest we:
1. Do it in a separate PR ;)
2. modify all of these to take/return the `rest.Config` _xor_ `rest.Interface` as arguments.  If the caller wants to get a `rest.Client` from a `Config` then they can call `RESTClientFor` directly.
3. Use `rest.InClusterConfig()` or similar directly in the `cmd/*` and avoid creating two families of all the variants here.
4. Reduce/rename *this* function to *just* configuring the client for accessing Function CRD resources.

Reply:
will do in a different PR